LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -983,7 +983,7 @@
         dialog.setSaveHandler(function() {
             addStep(currentWorkflow, $('#stepTitle').val(), function(stepID) {
                 if (isNaN(stepID)) {
-                    console.log(stepID);
+                    alert(stepID);
```

Comment:
Why are we doing an alert with the id?  Can the user do anything  with it?  Will the average user know what it means?

Reply:
'Restricted Command' is sent back and already alerted if some other built-in workflow modifications are attempted.  This one was only logged, which does not explain to the user why the step was not added.